web/src/lib/auth/signinHelper.test.ts

```diff
@@ -109,7 +109,7 @@ describe("SigninHelper", () => {
       expect(mockSetRegistrationStatus).toHaveBeenCalledWith("IN_PROGRESS");
     });
 
-    it("does not use a returnToLink if path is account-setup", async () => {
+    it("posts userData to api self-reg with current pathname included when returnToLink is empty", async () => {
```

Comment:
These test names were swapped for what they actually respectively test, so I just switched them.